mongo/integration/mtest/mongotest.go

```diff
@@ -521,7 +522,19 @@ func (t *T) ClearCollections() {
 			if coll.CreateOpts != nil && coll.CreateOpts.EncryptedFields != nil {
 				DropEncryptedCollection(t, coll.created, coll.CreateOpts.EncryptedFields)
 			}
-			_ = coll.created.Drop(context.Background())
+
+			c := coll.created
+			err := c.Drop(context.Background())
```

Comment:
Can we just do `err := coll.created.Drop(context.Background())` and then use a new variable on L534?

Reply:
Yep, that's fine. This was because I was initially asserting that an error doesn't happen but that's quite a different problem.